production/scrapers/montana.R

```diff
@@ -1,22 +1,18 @@
 source("./R/generic_scraper.R")
 source("./R/utilities.R")
 
-montana_check_date <- function(x, date = Sys.Date()){
-    base_html <- xml2::read_html(x)
-    date_check_txt <- rvest::html_nodes(base_html, xpath = "//*[@id=\"content-wrapper\"]/main/div/div[2]/div/p[4]/em/text()") %>%
-        rvest::html_text()
-
-    date_check_txt %>%
-        {.[str_detect(., "(?i)21")]} %>%
-        str_split("Data last updated at") %>%
-        unlist() %>%
-        .[2] %>%
+montana_check_date <- function(url, date = Sys.Date()){
+    base_html <- xml2::read_html(url)
+    date_check_txt <- rvest::html_nodes(base_html, css = 'em') %>%
+        rvest::html_text() %>% 
         str_split("on") %>%
         unlist() %>%
         .[2] %>%
-        str_remove("\\.") %>%
-        str_replace("Sept", "Sep") %>% 
-        lubridate::mdy() %>%
+        str_replace("Sept", "Sep")
+           
+    date_check_txt <- gsub("[[:punct:]]", "", date_check_txt)
+
+    date_check <- lubridate::mdy(date_check_txt) %>%
         error_on_date(expected_date = date)
```

Comment:
This looks great, though one issue is that the last line in this function is an assignment rather than a pipe out or an explicit return statement. I don't think the date check will register in the way we want (though I could be wrong, perhaps the default behavior is to use the last object assigned in a function?). In any case, I suggest adding one line below line 16:

```return(date_check)```

Reply:
I've found that R has implicit returns, something that's backed up by [this stack answer](https://stackoverflow.com/questions/11738823/explicitly-calling-return-in-a-function-or-not). ("the function() itself returns last evaluated value even without including return() function.") And because I haven't seen as many return statements in the code base, I thought that was more in keeping with our style in this repo. That said, I'll plan to add the return statement here because I think you're right that an explicit return can be clearer and easier to read.